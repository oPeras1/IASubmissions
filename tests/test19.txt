181	218	218	218	218	224	224	224	224	201	215	215	215	207	207	207	219	219	207	207	220	220	220	220	216	221	221	221	221	222	222	222	222	222	222	222	223	223	223	200
181	218	201	201	218	201	201	201	201	201	215	205	215	213	213	207	207	219	219	207	207	220	203	216	216	221	209	209	209	210	217	217	217	217	217	222	222	222	223	200
181	181	204	201	201	201	212	212	212	212	215	205	213	213	213	207	207	207	207	207	208	208	203	216	216	216	209	197	210	210	210	217	211	211	214	214	214	214	200	200
181	181	204	204	201	201	202	212	196	212	212	205	205	205	213	206	206	206	206	207	208	203	203	203	209	209	209	197	197	210	198	198	198	211	199	200	200	214	200	189
190	181	204	204	204	201	202	202	196	196	191	191	205	205	192	206	207	207	207	207	208	203	203	185	185	209	209	197	195	210	170	198	211	211	199	199	200	200	200	189
190	181	182	182	183	201	183	202	202	196	196	191	192	192	192	184	184	184	193	194	194	203	194	185	185	197	197	197	195	170	170	198	198	198	199	199	199	200	189	189
190	181	182	182	183	183	183	183	174	196	191	191	192	169	192	192	184	184	193	193	194	194	194	185	197	197	197	186	195	195	170	198	198	187	188	199	200	200	189	180
190	181	181	182	182	173	183	183	174	191	191	192	192	169	169	192	192	184	193	177	177	194	185	185	185	185	186	186	186	195	170	187	187	187	188	188	189	189	189	180
181	181	181	172	182	173	173	183	174	175	175	175	175	175	169	176	184	184	184	184	177	177	177	178	185	186	186	186	186	170	170	187	187	188	188	188	188	188	189	180
163	172	172	172	163	173	163	163	174	163	175	166	169	169	169	176	176	176	140	140	177	164	178	178	178	164	179	179	179	179	170	170	168	171	171	171	171	180	180	180
163	163	163	163	163	163	163	163	163	163	166	166	166	169	140	140	140	140	140	140	140	164	164	164	164	164	164	164	164	164	164	170	168	171	165	165	171	160	160	160
125	163	163	162	162	162	162	155	155	163	163	166	156	140	140	140	157	158	158	158	142	142	159	164	164	164	167	167	167	167	167	168	168	168	165	165	160	160	161	161
125	138	138	138	138	138	162	147	155	163	163	156	156	140	157	157	157	157	158	142	142	159	159	159	164	151	151	151	151	151	152	152	153	153	165	154	154	160	160	161
125	138	150	150	150	146	146	147	155	156	156	156	140	140	148	148	157	158	158	149	142	159	159	129	151	151	144	144	144	152	152	152	153	153	153	154	154	154	160	161
125	138	138	150	146	146	133	147	147	147	139	139	140	134	134	148	148	148	149	149	142	142	143	129	129	151	151	136	144	144	152	153	153	153	153	123	123	154	154	145
125	138	138	138	138	146	133	133	147	147	139	140	140	134	141	141	148	148	141	149	142	143	143	143	129	129	136	136	136	144	137	137	137	137	123	123	132	132	145	145
125	125	125	119	138	126	133	133	133	139	139	139	140	134	141	141	141	141	141	142	142	143	143	135	129	129	129	130	136	144	144	137	131	131	123	123	132	124	124	145
113	125	113	119	119	126	126	126	133	133	133	127	127	134	134	120	128	128	128	116	116	135	135	135	129	130	130	130	136	122	122	137	137	131	131	123	132	123	124	124
113	125	113	119	114	114	126	126	126	115	115	127	127	127	120	120	128	128	128	116	116	121	121	129	129	130	117	130	130	118	122	122	131	131	122	123	132	123	123	124
113	113	113	119	100	114	114	114	114	100	115	115	115	105	105	120	111	111	116	116	121	121	121	117	117	117	117	118	118	118	118	122	122	122	122	123	123	123	110	124
98	98	113	99	100	114	114	100	100	100	104	115	105	105	105	111	111	111	116	101	107	107	108	117	108	117	117	109	118	102	102	112	112	112	112	112	112	112	110	110
98	99	99	99	100	100	100	100	100	100	104	104	105	105	111	111	106	106	106	101	107	108	108	108	108	108	108	109	109	102	112	112	96	96	110	110	110	110	110	103
98	99	99	92	92	93	93	93	93	100	100	104	105	105	95	80	106	106	101	101	107	107	107	107	107	108	108	109	102	102	96	96	96	96	97	110	97	97	103	103
98	91	99	91	92	92	72	72	93	100	94	94	94	95	95	80	80	101	101	101	101	87	87	88	77	82	82	102	102	102	102	96	90	96	97	97	97	84	84	103
61	91	91	91	92	72	72	72	93	93	93	94	95	95	80	80	81	86	86	86	87	87	87	88	77	77	82	89	89	89	90	90	90	96	97	97	84	84	84	85
61	70	76	76	76	76	76	72	72	57	57	57	73	74	74	80	81	81	86	87	87	75	88	88	77	77	82	82	89	90	90	90	90	83	83	83	84	85	85	85
61	70	70	76	71	76	76	76	72	63	57	57	73	73	74	80	81	81	81	81	75	75	75	75	77	77	82	82	58	78	78	78	78	83	83	84	84	85	85	85
61	70	76	76	71	71	56	56	72	63	57	57	73	74	74	74	64	65	65	75	75	66	66	66	66	77	67	68	58	58	58	78	78	78	59	59	79	79	79	79
61	70	70	62	71	71	71	56	72	63	57	73	73	57	74	74	64	65	65	65	75	75	66	67	67	67	67	68	68	58	69	69	69	59	59	59	59	59	59	59
61	61	62	62	62	45	45	56	39	63	57	57	57	57	54	54	64	64	35	65	65	42	66	67	67	42	68	68	58	58	69	25	51	59	60	60	60	60	59	52
45	45	45	45	45	45	45	56	39	53	53	53	57	54	54	54	35	35	35	55	55	42	42	42	42	42	42	42	58	58	25	25	51	59	44	44	44	60	59	52
45	45	45	45	45	45	46	39	39	33	33	53	53	54	54	35	35	55	55	55	10	42	42	42	48	42	42	14	14	25	25	51	51	51	51	49	44	44	52	52
45	50	50	50	50	46	46	46	39	33	34	34	47	47	47	35	35	41	41	10	10	10	28	42	48	48	14	14	14	37	25	43	51	49	49	49	44	44	44	52
45	31	31	32	32	46	46	46	39	33	33	34	47	40	47	40	35	35	41	41	10	10	28	42	48	36	36	14	14	37	25	43	43	49	49	49	44	38	44	18
31	31	31	32	21	27	27	27	39	33	34	34	7	40	40	40	40	35	22	41	41	10	28	42	29	24	36	36	14	37	25	43	26	30	30	30	44	38	18	18
31	21	32	32	21	21	27	27	3	33	6	34	7	7	22	22	22	35	22	23	23	10	28	28	29	24	24	36	14	37	25	25	26	26	30	30	38	38	18	18
20	21	21	21	21	21	27	3	3	6	6	6	6	7	7	22	22	22	22	23	10	10	11	28	29	29	24	14	14	14	25	26	26	26	26	30	19	19	17	18
20	20	20	21	21	2	2	3	3	3	5	5	6	7	7	8	22	8	23	23	10	11	11	12	24	24	24	24	24	14	25	26	16	16	26	19	19	19	17	18
1	1	1	2	2	2	3	3	4	5	5	6	6	7	7	8	8	8	9	10	10	11	11	12	13	13	14	14	14	14	14	15	15	16	16	19	17	17	17	18
1	1	1	1	2	3	3	4	4	4	5	6	6	7	7	8	9	9	9	9	10	10	11	12	12	13	13	13	13	14	15	15	16	16	16	16	17	18	18	18