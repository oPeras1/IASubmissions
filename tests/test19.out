181	218	L	L	L	I	I	I	I	201	L	L	215	L	L	L	S	S	207	207	T	T	T	220	T	I	I	I	I	222	222	222	I	I	I	I	L	L	L	200
T	218	201	201	L	201	201	201	201	201	L	205	215	S	213	L	207	S	S	207	207	T	203	T	T	221	209	209	209	I	T	T	T	217	217	222	222	222	L	200
T	T	L	T	T	T	I	I	I	I	L	S	213	S	S	207	207	207	207	207	L	L	T	216	T	216	209	T	210	I	210	T	211	L	I	I	I	I	200	200
T	181	L	204	T	201	S	212	196	212	212	S	S	205	S	I	I	I	I	207	L	203	T	T	S	S	209	T	T	I	198	I	198	L	199	200	200	214	S	189
I	181	L	L	204	201	S	S	L	L	191	191	S	205	L	206	207	207	207	207	L	203	T	185	185	S	S	T	195	I	170	I	L	L	199	T	200	S	S	189
I	181	S	182	L	201	183	S	202	L	196	S	L	L	L	S	S	184	T	194	194	203	L	185	185	197	197	197	S	170	170	I	198	198	T	T	T	S	189	189
I	181	S	S	L	L	L	183	I	L	S	S	192	S	192	192	S	S	T	T	L	L	L	185	197	197	197	T	S	S	170	I	198	187	S	199	200	200	L	180
I	181	181	S	182	T	183	183	I	191	S	192	192	S	S	192	192	184	T	177	T	194	I	I	I	I	T	T	186	S	L	T	T	T	S	S	L	L	L	180
181	181	181	L	182	T	T	183	I	175	L	L	L	175	S	L	184	184	184	184	T	T	177	T	185	186	186	T	186	170	L	187	T	188	188	S	188	188	189	L
163	L	L	L	163	T	163	163	I	163	L	166	169	169	169	L	L	L	140	140	T	164	T	T	T	164	I	I	I	I	L	L	168	I	I	I	I	L	L	L
S	S	163	163	163	163	163	163	163	163	T	T	T	169	S	S	140	140	140	140	140	164	164	164	L	L	L	164	164	164	164	170	T	171	165	S	171	T	160	160
125	S	S	I	I	I	I	L	L	163	163	T	156	S	S	140	L	T	T	T	L	L	T	164	L	164	I	I	I	I	167	T	T	T	S	S	T	T	L	L
125	I	138	138	138	138	162	147	L	163	163	S	S	140	L	L	L	157	T	142	L	159	T	T	164	S	S	151	151	151	L	L	153	153	S	154	154	T	160	L
125	I	T	T	T	L	L	147	L	156	S	S	140	140	148	S	157	158	158	T	L	159	T	129	S	S	144	S	S	152	L	152	T	T	T	L	L	L	160	L
125	I	138	T	146	L	133	T	T	T	L	139	140	L	L	S	S	148	T	T	142	142	S	129	T	151	151	136	S	S	L	153	153	T	153	S	123	L	154	T
125	I	138	138	138	L	133	133	T	147	L	140	140	L	141	141	S	148	141	T	142	S	S	143	T	T	L	L	L	144	T	T	T	137	S	S	I	132	T	T
T	T	T	I	138	S	133	S	S	139	L	L	140	L	I	I	I	I	141	142	142	S	143	L	T	129	129	130	L	144	144	T	131	L	S	123	I	124	124	T
113	T	113	I	119	S	S	126	S	S	133	S	S	134	134	T	128	S	S	L	116	L	L	L	129	L	L	L	136	122	S	137	137	L	131	123	I	123	L	L
L	125	113	I	114	114	S	126	126	115	L	127	S	S	T	T	S	S	128	L	116	T	121	129	129	L	117	130	130	L	S	S	L	L	122	123	I	123	123	L
L	L	L	I	100	L	L	L	114	100	L	L	L	105	105	T	111	111	L	L	T	T	T	I	I	I	I	L	L	L	118	S	122	122	122	123	123	123	110	L
I	98	113	L	100	L	114	T	T	T	S	115	T	T	T	L	L	L	116	I	107	107	108	117	L	117	117	T	118	I	102	L	L	L	112	112	112	112	110	110
I	L	L	L	100	100	100	100	T	100	S	S	105	T	111	L	106	S	S	I	107	108	108	108	L	L	L	T	T	I	112	L	96	96	I	I	I	I	110	T
I	99	99	S	S	93	L	L	L	100	100	S	105	105	S	80	S	S	101	I	107	I	I	I	I	108	108	T	102	I	T	T	T	96	T	110	97	97	T	T
I	L	99	91	S	S	72	72	L	100	T	T	T	S	S	L	L	101	101	I	101	S	87	L	77	L	L	102	102	I	102	T	90	96	T	T	97	S	84	T
61	L	L	L	92	72	72	S	93	93	93	T	95	S	80	L	81	T	T	T	S	S	87	L	77	77	L	T	T	T	L	L	L	96	T	97	S	S	84	S
61	T	76	I	I	I	I	S	S	57	57	57	T	74	I	L	81	81	T	87	S	75	L	L	77	T	L	82	T	90	L	90	90	L	L	L	S	85	S	S
61	T	T	76	S	76	76	76	S	I	57	57	T	T	I	80	I	I	I	I	L	L	75	75	T	T	82	82	S	78	T	T	T	L	83	84	84	85	S	85
L	T	76	76	S	S	L	L	72	I	57	57	T	74	I	74	L	65	S	75	L	66	L	L	66	T	67	L	S	S	58	T	78	78	S	S	I	I	I	I
L	70	70	T	71	S	71	L	72	I	57	73	73	L	I	74	L	65	S	S	L	75	L	67	S	S	67	L	68	S	L	L	L	S	S	59	59	59	59	59
L	L	T	T	T	45	45	L	39	I	57	L	L	L	54	54	L	L	35	S	65	42	L	S	S	42	L	L	58	58	L	25	51	59	I	I	I	I	59	52
45	45	45	45	L	L	L	56	T	53	S	S	57	T	T	T	I	35	35	L	55	S	S	42	42	42	42	42	58	58	25	25	51	59	44	44	S	60	59	T
45	45	45	45	L	45	S	T	T	33	33	S	S	54	T	35	I	L	L	L	T	42	S	S	T	42	42	14	L	25	25	I	I	I	I	L	S	S	T	T
45	I	I	I	I	S	S	46	T	I	34	34	L	L	L	35	I	41	S	10	T	T	I	42	T	T	L	L	L	I	25	T	51	49	49	L	44	S	44	T
45	S	31	L	32	S	46	46	39	I	33	T	L	40	47	L	I	35	S	S	T	10	I	42	T	36	S	14	14	I	25	T	T	49	L	L	44	L	44	I
S	S	31	L	21	L	L	L	39	I	T	T	7	L	L	L	40	35	22	S	41	10	I	42	L	24	S	S	14	I	25	T	26	S	S	30	44	L	18	I
S	21	L	L	21	S	27	L	3	I	6	T	7	S	22	S	S	35	22	L	23	10	I	28	L	24	24	S	14	I	L	L	26	26	S	S	L	L	18	I
L	21	21	21	S	S	27	S	S	6	6	6	I	S	S	22	S	S	22	L	10	10	S	28	L	L	24	14	14	14	L	26	L	L	L	30	S	19	L	I
L	L	L	21	S	2	2	3	S	S	L	L	I	7	S	T	22	8	L	L	10	S	S	L	24	I	I	I	I	14	L	26	16	16	L	S	S	19	L	18
1	T	1	T	T	T	3	3	T	5	L	6	I	7	7	T	T	8	T	10	10	S	11	L	13	13	14	14	14	14	14	S	S	L	16	S	17	L	L	18
T	T	T	1	T	3	3	T	T	T	L	6	I	7	7	T	9	T	T	T	10	10	11	L	L	I	I	I	I	14	S	S	16	L	L	L	17	18	18	18